catalog.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="ru" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Перфораторы</title>
+  </head>
+  <body>
+    <header>
+      <p class="titel-shop">ТЕХНОМАРТ</p>
```

Comment:
проверь по всему проекту, чтобы названия классов были без опечаток
```suggestion
      <p class="title-shop">ТЕХНОМАРТ</p>
```